actors/builtin/miner/miner_state.go

```diff
@@ -921,6 +922,26 @@ func (st *State) SaveDeadlines(store adt.Store, deadlines *Deadlines) error {
 	return nil
 }
 
+// LoadVestingFunds loads the vesting funds table from the store
+func (st *State) LoadVestingFunds(store adt.Store) (*VestingFunds, error) {
+	var funds VestingFunds
+	if err := store.Get(store.Context(), st.VestingFunds, &funds); err != nil {
+		return nil, xc.ErrIllegalState.Wrapf("failed to load vesting funds (%s): %w", st.VestingFunds, err)
```

Comment:
There's no need to specify `ErrIllegalState` explicitly, that's the assumed default unless we make some giant codebase-wide change of policy. Just use `xerrors`.